lib/scope.js

```diff
@@ -628,22 +630,18 @@ module.exports = {
     return selector;
   },  // Ends scope.getSelector()
 
-  getPossibleTableRowMatchers: async function ( scope, { node, var_names, values }) {
+  getPossibleTableRowMatchers: async function ( scope, { var_names, values }) {
     /* Return variable names and values combined in ways that can
-    * try to match table rows. Tables can have three properties:
-    * variable name, value, and checked (when appropriate, whether
-    * the element should be checked or not). `checked` here will
-    * let us know if we need to change the state of the element. */
+    * try to match table rows. Tables can have two properties:
+    * variable name and value. `checked` will be removed from here
+    * as it'll have to be reevaluated later anyway. */
     let possible_table_rows = [];
     for ( let var_name of var_names ) {
       // Inputs like textareas don't have values
       if ( values.length > 0 ) {
         for ( let value of values ){
-          // Get the status of elements that can be checked or unchecked
-          // In HTML, if `checked` is absent, it is considered `false`
-          // Extra ones that get caught in here shouldn't matter...
-          let row = { var: var_name, value, checked: false };
-          if ( node.attribs.checked !== undefined ) { row.checked = node.attribs.checked; }
+          // Prepare to remove `checked`
+          let row = { var: var_name, value, checked: '' };
```

Comment:
I was making sure that it was fine without the usual values while at the same time not getting rid of it. That is, that its lack of useful value would not interfere with current functionality. There will be several changes to make to get rid of it completely, so I was holding off on that. If we were keeping it around, it would not be an empty string.